Libraries/CameraRoll/RCTImagePickerManager.m

```diff
@@ -101,9 +104,23 @@ - (dispatch_queue_t)methodQueue
 - (void)imagePickerController:(UIImagePickerController *)picker
 didFinishPickingMediaWithInfo:(NSDictionary<NSString *, id> *)info
 {
-  [self _dismissPicker:picker args:@[
-    [info[UIImagePickerControllerReferenceURL] absoluteString]
-  ]];
+  // Image from PhotoLibrary
+  NSString *imageUri = [info[UIImagePickerControllerReferenceURL] absoluteString];
+  if (imageUri) {
+    [self _dismissPicker:picker args:@[imageUri]];
+
+  } else {
+    // Image from CameraRoll hasn't uri.
+    // We need to save it to the store first.
+    UIImage *originalImage = (UIImage *) [info objectForKey: UIImagePickerControllerOriginalImage];
+    [_bridge.imageStoreManager storeImage:originalImage withBlock:^(NSString *tempImageTag) {
```

Comment:
Can you add a warning here that this will cause a memory-leak as-is? You need to remove the image from the imageStoreManager once you're done using. We're looking into providing automatic solutions for this.